pkg/schema/error.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package schema
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/k14s/ytt/pkg/filepos"
+	"github.com/k14s/ytt/pkg/yamlmeta"
+)
+
+type errorWithContent struct {
+	contents string
+	Position *filepos.Position
+}
+
+func (e errorWithContent) Error() string {
```

Comment:
When deleting this function were found that all the tests still pass and there were no compilation errors. Is the purpose of this function just to make "errorWithContent" adhere to the error interface? Why is this here?

Reply:
Removed alongside the struct.